mmaction/datasets/charades_dataset.py

```diff
@@ -0,0 +1,120 @@
+import copy
+import os.path as osp
+from collections import defaultdict
+
+import numpy as np
+
+from .base import BaseDataset
+from .registry import DATASETS
+
+
+@DATASETS.register_module()
+class CharadesDataset(BaseDataset):
+
+    def __init__(self,
+                 ann_file,
```

Comment:
you can use  `*args` and `**kwargs` for simplity

Reply:
Some parameters are not supported by charadesdataset, and we need to modify the default value of multi_class, num_classes. So, maybe better to keep it as it is